internal/converter/testdata/proto/JSONFields.proto

```diff
@@ -0,0 +1,10 @@
+syntax = "proto3";
+package samples;
+
+message JSONFields {
+    string name       = 1;
+    string timestamp  = 2 [json_name="time_stamp"];
```

Comment:
Ah, I see the json name is purposefully snake_case.

Can you add a field that isn't snake_case like `string message_id = 6`?

Reply:
OK cleaned it up